### PR TITLE
Hide student corporation forms from timeline

### PR DESCRIPTION
**Pull request by maggicl**
They're kind of annoying, especially if you have a lot of assignments

### CODE REVIEW

**Comment by Steeven9** [comment]
I mean it's enough to unenroll from the student corporation course to make everything go away

**Comment by warman69** [comment]
I'm 😳🥴 sorry, 😕 but 🍑 we 👧 can't 💦👆 work 💼🔛 together 👨‍👦 if you 👈 don't 🙅🙈 want 🤲🤲 to follow 👣 the rulers 🚷 of the Corporation. The rectorate cares 🤷‍♂️ and manages the university environment for 📷 the benefit of the students 📕📖 and, not 👎 just 👽 themselves. 🥺💦💦 One 🔥 of, if not 🚫 the safest campuses in 😀🔥 the Switzerland. 🇨🇭 Corporation cares 🤷 about 💭 us 👨🏿‍💻 students 👨‍🎓🧑‍🎓👩‍🎓 and your 👉 request 🙏🏼🙌🏼 seems 👀 to be ⬇️ an unconscious 😵 principle 👴🏾 of mutiny 🏴‍☠️. I 👥 can 💦💦 not 😳 allow 👨‍👨‍👦‍👦🎫🎫 you 🦎 to bear 🍯 such 😆 bad ⛔ perspective on 😼 my 😊👼 glorious corporation. I 😊 will 🙏👊 never 🙅 betray may 🐵 country! 🇨🇭🇨🇭🇨🇭🇨🇭🇨🇭 The Switzerland 🇨🇭 protects democracy 🗽🇺🇸🏈 and libertarian values, 💰👪 the Corporation is sacred under 😡 the right 📷 of the Swiss Constitution. 📜 🇨🇭🇨🇭🇨🇭🇨🇭🇨🇭

**Comment by maggicl** [comment]
Thank you @warman69 (Martin Donati) for your insightful comment

**Comment by tommi27** [comment]
Nice argument @warman69, why don't you back it up with a source?

**Comment by maggicl** [comment]
> I mean it's enough to unenroll from the student corporation course to make everything go away

BTW @Steeven9 you could find yourself wanting to be a member of the _Student corporation_ course because, you never know, you might want to submit a proposal sometime. But it's very annoying to see those submissions as deadlines in the timeline, it freaks me out everytime i open iCorsi

**Comment by warman69** [comment]
> Nice argument @warman69, why don't you back it up with a source?

My 🙋🏻‍♀️ source 🚨😔 is that 👑 I 👁️ made 🤑👑 it the fuck 🤬🤬🤬 up 🍅

**Comment by warman69** [comment]
> BTW @Steeven9 you could find yourself wanting to be a member of the _Student corporation_ course because, you never know, you might want to submit a proposal sometime. But it's very annoying to see those submissions as deadlines in the timeline, it freaks me out everytime i open iCorsi

Every 💯 one 😤🏼 wants 💩 to be 🍆 part 💔 of the Corporation. It is a privilege ✊ for 🤤 us 🇺🇸👨 Swiss students 📕📖 paying 😏💰🤑 2000 CHF (SWISS FRANKS) per semester! 🍁

**Comment by Steeven9** [comment]
_Understandable, have a nice day_

**Comment by warman69** [comment]
> _Understandable, have a nice day_

Thanks, have a nice weekend to you too!

File: icorsi.html [hunk 1]
```diff
@@ -9,6 +9,21 @@
   studyYear: '2',   // CCC
 };
 
+const interval = setInterval(() => {
+  const cantoni = ["Modulo per lanciare", "Form to launch"]
```
Comment (Steeven9) [inline icorsi.html:13]: Why are the Swiss cantons mentioned in code?

Comment (warman69) [reply icorsi.html:13]: They're 🐂 not 😐 the Swiss [cantons](https://www.youtube.com/watch?v=dQw4w9WgXcQ) but 🧈 the Swiss [Cantoni](https://search.usi.ch/it/persone/c3153bd11a31b6f96024b89b182d629f/cantoni-lorenzo)!

**Pull request closed without merging**